lib/types/meta_data.ex

```diff
@@ -26,7 +26,15 @@ defmodule Kadena.Types.MetaData do
           chain_id: chain_id()
         }
 
-  defstruct [:creation_time, :ttl, :gas_limit, :gas_price, :sender, :chain_id]
+  defstruct creation_time: 0,
+            ttl: 0,
+            gas_limit: 0,
+            gas_price: 0,
+            sender: "",
+            chain_id: %ChainID{id: "0"}
+
+  @impl true
+  def new, do: %__MODULE__{}
 
   @impl true
   def new(args) when is_list(args) do
```

Comment:
@Odraxs @EdwinGuayacan This is not really necessary, you can use a default value for the arguments

```elixir
def new (args \\ [])
def new(args) when is_list(args) do
  ...
end
```